include/Engine/Network/Server.h

```diff
@@ -32,6 +33,7 @@ class Server : public Network
     // Network channels
     TCPServer m_Reliable;
     UDPServer m_Unreliable;
+    UDPServer m_ServerlistRequest;
     // dont forget to set these in the childrens receive logic
     boost::asio::ip::address m_Address;
     int m_Port = 27666;
```

Comment:
This will always be set to the value in the config file. This is merely a default value.